QuantConnect.DataBento/DataBentoDataProvider.cs

```diff
@@ -422,7 +352,7 @@ private void OnDataReceived(object? sender, BaseData data)
                 {
                     tick.Time = GetTickTime(tick.Symbol, tick.Time);
```

Comment:
The greatest way to convert time internally when creating an instance of the Tick.